pkg/network/netlink/consumer.go

```diff
@@ -0,0 +1,404 @@
+package netlink
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+	"sync/atomic"
+	"syscall"
+
+	"github.com/pkg/errors"
+
+	"github.com/DataDog/datadog-agent/pkg/process/util"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"golang.org/x/sys/unix"
+
+	"github.com/mdlayher/netlink"
+	"github.com/mdlayher/netlink/nlenc"
+)
+
+const (
+	netlinkCtNew      = uint32(1)
```

Comment:
nit, will you please add some documentation to these values?

Reply:
Sure 👍

Reply:
This is done.